src/ser.rs

```diff
@@ -215,7 +215,7 @@ impl ser::SerializeStruct for ObjectSerializer<'_> {
 #[derive(Default)]
 pub struct Serializer {
     serialize_maps_as_objects: bool,
-    serialize_64_bit_numbers_as_big_int: bool,
+    serialize_large_number_types_as_big_ints: bool,
```

Comment:
Also nitpicking, but JS uses `bigint` for primitive type, so I think we should write them together in the API too.